src/harness.js

```diff
@@ -168,7 +168,12 @@ class Harness {
    */
   async clear() {
     clearSync(this);
-    return this.page && await this.page.evaluate(() => window.restoreTimers());
+    if(!this.page) {
+      return Promise.resolve();
+    }
+
+    await this.page.reload();
```

Comment:
would you add a comment to explain why we reload. or link to the issue or conversation about it? just a bit of a paper trail here might be nice because it isn't obvious why this is required in harness@3.x and not 2.x

Reply:
Absolutely!  I have promoted the original draft issue to https://github.com/medic/cht-conf-test-harness/issues/185 where we can reference it later (and included a comment in the code about it).